consensus/polybft/wallet/account.go

```diff
@@ -110,7 +110,12 @@ func (a *Account) Save(secretsManager secrets.SecretsManager) (err error) {
 }
 
 func (a *Account) GetEcdsaPrivateKey() (*ecdsa.PrivateKey, error) {
-	ecdsaRaw, err := a.Ecdsa.MarshallPrivateKey()
+	return AdaptECDSAPrivKey(a.Ecdsa)
+}
+
+// AdaptECDSAPrivKey converts ecdsa private key from wallet.Key to ecdsa.PrivateKey instance
```

Comment:
I've forgotten to update the comment, let's change it now.

```suggestion
// AdaptECDSAPrivKey converts ecdsa private key from crypto.ECDSAKey to ecdsa.PrivateKey instance
```